brave/brave-core
# Add weekly Wallet transaction sent P3A metric

Resolves brave/brave-browser#26224

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0. Start browser with `--p3a-count-wallet-test-networks` flag. Ensure profile contains wallet with testnet tokens.
1. Ensure metric does not exist in local state.
2. Send transaction, ensure metric has value of 1.
3. Close browser, move system time ahead by seven days.
4. Open browser, ensure metric has value of 0.
5. Send another transaction, ensure metric has value of 1.
